pegen/pegen.c

```diff
@@ -609,7 +602,7 @@ run_parser_from_file(const char *filename, void *(start_rule_func)(Parser *), in
     tok->filename = filename_ob;
     filename_ob = NULL;
 
-    result = run_parser(tok, start_rule_func, mode, keywords, n_keyword_lists);
+    result = run_parser(tok, start_rule_func, mode, 0, keywords, n_keyword_lists);
```

Comment:
Suggestion: Let's use an enum for the 1 and 0 values so we know what they mean when reading the code